client/db/types.go

```diff
@@ -99,21 +111,25 @@ func DecodeAccountInfo(b []byte) (*AccountInfo, error) {
 }
 
 func decodeAccountInfo_v0(pushes [][]byte) (*AccountInfo, error) {
-	if len(pushes) != 5 {
-		return nil, fmt.Errorf("decodeAccountInfo: expected 5 data pushes, got %d", len(pushes))
+	if len(pushes) != 7 {
+		return nil, fmt.Errorf("decodeAccountInfo: expected 7 data pushes, got %d", len(pushes))
 	}
 	hostB, keyB, dexB := pushes[0], pushes[1], pushes[2]
 	coinB, certB := pushes[3], pushes[4]
 	pk, err := secp256k1.ParsePubKey(dexB)
 	if err != nil {
 		return nil, err
 	}
+	var id [blake256.Size]byte
+	copy(id[:], pushes[5])
 	return &AccountInfo{
 		Host:      string(hostB),
 		EncKey:    keyB,
 		DEXPubKey: pk,
 		FeeCoin:   coinB,
 		Cert:      certB,
+		ID:        id,
+		Disabled:  bytes.Equal(pushes[6], encode.ByteTrue),
```

Comment:
This would give `false` if the length of `pushes[6]` was 2, or if `pushes[6][0]` were say 42.